src/app/ubs/ubs-admin/components/ubs-admin-order/ubs-admin-order.component.ts

```diff
@@ -582,8 +533,9 @@ export class UbsAdminOrderComponent implements OnInit, OnDestroy, AfterContentCh
       for (const formControlName in formItem.controls) {
         if (Object.prototype.hasOwnProperty.call(formItem.controls, formControlName)) {
           const formControl = formItem.controls[formControlName];
-
-          if (formControl instanceof FormControl) {
+          if (formControlName === 'userInfoDto' && formControl.dirty) {
```

Comment:
Repeated logic for type checks and value assignment is extracted into helper methods, reducing redundancy.
Centralize the logic for `dirty` checks, etc
`private isFormControl(item: AbstractControl): item is FormControl {
  return item instanceof FormControl;
}
`